app/src/main/java/org/oppia/android/app/devoptions/forcenetworktype/ForceNetworkTypeActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.oppia.android.app.devoptions.forcenetworktype
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.R
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** Activity for Force Network Type. */
```

Comment:
```suggestion
/** Activity for forcing the network mode for the app. */
```

Prefer being more detailed (& to be judicious about capitalizing words). Ditto for other KDocs in this PR.

Reply:
Done.